src/standard/statement.ml

```diff
@@ -498,6 +498,12 @@ let fun_def ?loc id vars params ret_ty body =
     def ?loc id ~vars ~params ret_ty body
   ]
 
+let pred_def ?loc id vars params ret_ty body =
```

Comment:
Is `ret_ty` really useful here ? given that predicate should have a return type of `bool`